deployments/package-lock.json

```diff
@@ -0,0 +1,9567 @@
+{
+  "name": "deployments",
+  "version": "1.0.0",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "deployments",
+      "version": "1.0.0",
+      "license": "MPL-2.0",
+      "dependencies": {
+        "@cdktf/provider-aws": "16.0.3",
```

Comment:
I'd expect aws and azurerm providers to be obsolete (not required for this PR). Please remove form here as well as the sbom and 3rd-party licenses if I am correct

Reply:
removed